spec/factories/person.rb

```diff
@@ -2,8 +2,19 @@
 
 FactoryBot.define do
   factory :person do
+    association(:ethnicity)
+    association(:gender)
+
     profiles { build_list :profile, 1 }
 
+    first_names { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    date_of_birth { Date.new(1980, 10, 20) }
+
+    police_national_computer { 'AB/1234567' }
```

Comment:
it is worth using faker here as well for those three identifiers, as those ideally should be unique  in further work?

Reply:
Yeah, I'll use a random string sequence matching the number of chars for each identifier. Doesn't have to be perfect :) - just needs to be unique.